### DISCUSSION TITLE
One shots and standalone loops in Grid Mode

### DISCUSSION THREAD

**Discussion by Phonowav** [SynthstromAudible/DelugeFirmware#1383] [Ideas]
Assign oneshots and standalone loops to empty pads in Grid Mode — basically like SP404 oneshot mode where oneshots can also be set to loop (even if song is not set to play).

**Comment by m-m-adams** [comment]
Unfortunately this is very difficult with how loops and clips tie into deluge playback. I'd say it's unlikely in the near future and the closest you'll get right now is using a kit
